fed/mosaic-environment/src/main/resources/CEnvironmentScheme.json

```diff
@@ -1,94 +1,165 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "CEnvironmentScheme",
+    "description": "Schema describing the JSON file structure for the environment configuration. It is is used to define which events are evaluated by the simulator. Entities entering the areas of the events, are notified by the 'EnvironmentSensorUpdates' interaction",
     "type": "object",
     "properties": {
         "events": {
+            "description": "List of events.",
             "type": "array",
-            "items": [ { "$ref": "#/definitions/event" } ]
+            "items": { "$ref": "#/definitions/event" }
         }
     },
     "definitions": {
         "event": {
+            "title": "event",
+            "description": "Object to define a single event configuration.",
             "type": "object",
             "properties": {
-                "type": { "$ref": "#/definitions/eventType" },
-                "location": { "$ref": "#/definitions/eventLocation" },
-                "time": { "$ref": "#/definitions/eventTime" }
+                "type": { 
+                    "description": "Type of the event.",
+                    "$ref": "#/definitions/eventType" 
+                },
+                "location": { 
+                    "description": "The location of the event,, this can either be a geoArea or a string representing a specific road segment.",
+                    "$ref": "#/definitions/eventLocation" 
+                },
+                "time": { 
+                    "description": "Time of the event.",
+                    "$ref": "#/definitions/eventTime" 
+                }
             },
             "required": [ "type", "location", "time" ]
         },
         "eventType": {
+            "title": "eventType",
+            "description": "Object to define the type of an event.",
             "type": "object",
             "properties": {
-                "sensorType": { "type": "string" },
-                "value": { "type": "integer", "default": 1 }
+                "sensorType": { 
+                    "description": "Represents the type of sensor value this event is emitting (e.g. Ice, Snow, or an arbitrary Obstacle)",
+                    "type": "string" 
+                },
+                "value": { 
+                    "description": "Used for assigning a value to the event. It can be used as the strength of an event, the amount of free parking spots in a parking lot, etc.",
+                    "type": "integer", 
```

Comment:
I think, this should be `number`. Could you check that?

Reply:
And please check for boundaries as well.

Reply:
In the code it is defined as int. As for boundaries I can't derive anything directly from the code or doc. Is there supposed to be a boundary?

Reply:
Okay, it makes sense, that there don't exist any boundaries. But the integer type is strange. You can leave it like that, but take a note in your issue desciption that here a double would be better.